apps/src/aichat/types.ts

```diff
@@ -1,45 +1,52 @@
 import {LevelProperties} from '@cdo/apps/lab2/types';
 
-import {Role} from '../aiComponentLibrary/chatItems/types';
+import {Role} from '../aiComponentLibrary/chatMessage/types';
+
+export const ChatEventDescriptions = {
+  CLEAR_CHAT: 'The user clears the chat workspace.',
+  LOAD_LEVEL: 'The user loads the aichat level.',
+} as const;
 
 // TODO: Update this once https://codedotorg.atlassian.net/browse/CT-471 is resolved
 export type AichatInteractionStatusValue = string;
 
-export interface ChatItem {
+export interface ChatEvent {
   // UTC timestamp in milliseconds
   timestamp: number;
+  hideForParticipants?: true;
```

Comment:
Minor (as a follow-up) - let's add a comment here explaining why this field is optional but type `true` instead of boolean (doesn't have to defined, but if defined it must be true, i.e. we don't want `hideForParticipants: false`)

Reply:
Good call! Will do in this follow-up PR: https://github.com/code-dot-org/code-dot-org/pull/60110